com.ibm.wala.cast.java/build.gradle

```diff
@@ -4,16 +4,35 @@ plugins {
 
 eclipse.project.natures 'org.eclipse.pde.PluginNature'
 
-sourceSets.main.java.srcDirs = ['src']
-
 dependencies {
 	implementation(
 			project(':com.ibm.wala.cast'),
 			project(':com.ibm.wala.core'),
 			project(':com.ibm.wala.shrike'),
 			project(':com.ibm.wala.util'),
 	)
+	testFixturesImplementation(
+			'junit:junit:4.12',
```

Comment:
Didn't we bump to junit 4.13?

Reply:
Good catch. Fixed in 6ad2465ecfc698cc707c437218c645c0db093fec.